### PR TITLE
cardano: add support for tokens

### PR DESCRIPTION
**Pull request by benma**
Every transaction output can also have tokens attached to it.

Specification: https://github.com/input-output-hk/cardano-ledger/blob/80d2624174b3fc598c95ab9bf0a21c9bc7e045d3/eras/alonzo/test-suite/cddl-files/alonzo.cddl#L362

Every output containing tokens also needs to have a minimum amount of
ADA in it, accoding to
https://github.com/input-output-hk/cardano-ledger/blob/80d2624174b3fc598c95ab9bf0a21c9bc7e045d3/doc/explanations/min-utxo-alonzo.rst.

The format for making `asset...` identifiers:
https://github.com/cardano-foundation/CIPs/tree/ae83a0420be5485f7fe81cdc23d73cf6a237c16a/CIP-0014

General info about tokens:

https://docs.cardano.org/native-tokens/learn

https://github.com/input-output-hk/cardano-ledger/blob/80d2624174b3fc598c95ab9bf0a21c9bc7e045d3/doc/explanations/token-bundles.rst

### CODE REVIEW

**Comment by benma** [comment]
I tested this also by successfully broadcasting a token transfer tx on Adalite (the code changes for this are coming in separate PRs in the relevant repos).

File: py/send_message.py [hunk 1]
```diff
@@ -808,6 +808,58 @@ def sign_zero_ttl() -> None:
...
+                            ),
+                            prev_out_index=0,
+                        )
+                    ],
+                    outputs=[
+                        bitbox02.cardano.CardanoSignTransactionRequest.Output(
+                            encoded_address="addr1q9qfllpxg2vu4lq6rnpel4pvpp5xnv3kvvgtxk6k6wp4ff89xrhu8jnu3p33vnctc9eklee5dtykzyag5penc6dcmakqsqqgpt",
+                            value=1000000,
+                            asset_groups=[
+                                # Asset policy ids and asste names from: https://github.com/cardano-foundation/CIPs/blob/a2ef32d8a2b485fed7f6ffde2781dd58869ff511/CIP-0014/README.md#test-vectors
```
Comment (x1ddos) [inline py/send_message.py:829]: tiny typo: asste -> asset

File: messages/cardano.proto [hunk 2]
```diff
@@ -64,11 +64,24 @@ message CardanoSignTransactionRequest {
     uint32 prev_out_index = 3;
   }
 
+  // https://github.com/input-output-hk/cardano-ledger/blob/d0aa86ded0b973b09b629e5aa62aa1e71364d088/eras/alonzo/test-suite/cddl-files/alonzo.cddl#L358
+  message AssetGroup {
+    bytes policy_id = 1;
+
+    message Token {
+      bytes asset_name = 1;
+      uint64 value = 2;
```
Comment (x1ddos) [inline messages/cardano.proto:73]: would appreciate a comment clarifying this is a count of tokens, not an ADA value or a value of tokens in ADA or some other unit.

**Review by x1ddos — APPROVED** [review]
LGTM and quite clear :+1:

**Pull request merged**